src/backend/executor/nodeValuesscan.c

```diff
@@ -28,7 +28,8 @@
 #include "cdb/cdbvars.h"
 #include "executor/executor.h"
 #include "executor/nodeValuesscan.h"
-#include "optimizer/var.h"              /* CDB: contain_var_reference() */
```

Comment:
Have the above comment outdated?

Reply:
No, it has not. Not sure why it was changed.

Reply:
Fixed in https://github.com/arenadata/gpdb/pull/197/commits/6ee243024a4362b3f256d73991d8cf47b5e8a14f